### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ashiiidev**
please merge my pr
it will be your pleasure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,9 @@ We would love to know which resources you think deserve to be included in our li
 - [Treehouse](https://teamtreehouse.com/)
 - [Khan Academy](https://www.khanacademy.org/computing/computer-programming)
 - [W3Schools](https://www.w3schools.com/)
+- [treehouse](https://teamtreehouse.com/)
```
Comment (jdwilkin4) [inline README.md:36]: This is a duplicate.
Please remove this

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Ashiiidev** [comment]
I have updated my file and removed that link ne now please merge my pr

**Comment by Ashiiidev** [comment]
I changed my file now please merge into main

**Comment by Ashiiidev** [comment]
Thanks so much

**Pull request merged**